assets/js/googlesitekit/modules/create-info-store.test.js

```diff
@@ -44,9 +45,9 @@ describe( 'createInfoStore store', () => {
 
 	describe( 'storeName', () => {
 		it( 'returns the correct default store name', () => {
```

Comment:
This test should be updated to test that it throws an error if no `storeName` is provided since it is a required argument now (there is no default now).

Reply:
Marked as resolved but still unaddressed.

Reply:
I see you've added a new test to address the initial request here. In that case, this test should be removed since there is no default store name now.

Reply:
I've now [removed the default store name test](https://github.com/google/site-kit-wp/pull/2525/files#diff-734cdd487d9f3b1bb2aea8d73f6c8317c89712f1df2fec7a8141915bee9edf4fL46) in favour of testing passing and failing to pass the storeName.